fuzz/fuzz_targets/fuzz_common.rs

```diff
@@ -388,3 +391,24 @@ pub fn generate_random_string(max_length: usize) -> String {
 
     result
 }
+
+pub fn generate_random_file() -> Result<String, std::io::Error> {
+    let mut rng = rand::thread_rng();
+
+    let file_name: String = (0..10)
+        .map(|_| rng.sample(Uniform::new_inclusive(b'a', b'z')) as char)
+        .collect();
+    let mut file_path = temp_dir();
+    file_path.push(file_name);
+
+    let mut file = File::create(&file_path)?;
+
+    let content_length = rng.gen_range(10..1000);
+    let content: String = (0..content_length)
+        .map(|_| (rng.gen_range(b' '..b'~' + 1) as char))
```

Comment:
Shouldn't this also work?
```suggestion
        .map(|_| (rng.gen_range(b' '..=b'~') as char))
```